.github/workflows/host-image-ci.yml

```diff
@@ -185,6 +188,11 @@ jobs:
       with:
         name: u-boot
         path: "tools/cuttlefish-host-image-installer"
+    - name: Download docker-image-x86_64.tar
+      uses: actions/download-artifact@7a1cd3216ca9260cd8022db641d960b1db4d1be4 # aka v4.0.0
+      with:
+        name: docker-image-x86_64
+        path: "tools/cuttlefish-host-image-installer"
```

Comment:
Do you want to upload docker-image-arm64 as well?

If so, you need another action item similar to line 191-195 for ARM64.

Reply:
Yes, exactly..! I missed that since it was out of scope I can test. Thanks!

Reply:
Done.